### PR TITLE
3.15create forgot password page

### PR DESCRIPTION
**Pull request by Bagdunn**

### CODE REVIEW

File: client/src/pages/ResetPasswordPage/ResetPassword.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              className = {styles.inputPas}
+            />
+            <Button
+              variant='contained'
+              type='submit'
+              className = {styles.resetButton}
+            >
+              Reset password
+            </Button>
+            <a href='/login' className={styles.loginHref}>Back to log in</a>
```
Comment (Vadym342) [inline client/src/pages/ResetPasswordPage/ResetPassword.jsx]: you should to use Link

File: client/src/configs/routes.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ const routes = {
   NonAuthRoutes : {
     pathToLogin: '/',
     pathToRegistration: '/registration',
+    pathToReset: '/reset',
+    pathToForgotPassword: '/forgotpass',
```
Comment (MattMsh) [inline client/src/configs/routes.js]: Let's simplify it to: '/forgot'

File: client/src/pages/ForgotPasswordPage/ForgotPassword.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          </div>
+          <p className={styles.nameOfBox}>Forgot password?</p>
+          <p className={styles.requirementsText}>No disturbed, we`ll send instruction.</p>
+          <form className={styles.mainForm}>
+            <p className={styles.emailText}>Email</p>
+            <Input
+              name='Email'
+              type='Email'
+              placeholder='enter your Email'
+              className = {styles.inputPas}
```
Comment (MattMsh) [inline client/src/pages/ForgotPasswordPage/ForgotPassword.jsx]: Let's use style object for MUI components, like
```
const style = {
  key: value 
}
```

**Review by MattMsh — CHANGES_REQUESTED** [review]

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 4]
```diff
@@ -0,0 +1,128 @@
+@import '../../consts/variables.scss';
+
+.reset{
+  margin: 0;
+  padding: 0;
+  text-align: center;
+  &__title {
+    width: 100%;
+    height: 15%;
+    margin-top: 4rem;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: In all file sizes in px, here in rem

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+    }
+    & > p {
+      text-transform: uppercase;
+      font-weight: bold;
+      font-size: 18px;
+      margin-top: 0;
+    }
+  }
+
+.mainBox {
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss:30]: in Box must be radius 50px

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+      height: 100%;
+      width: 100%;
+      margin: auto;
+      border-radius: 50px;
+      position: relative;
+      box-shadow: $boxBgShadow;
+      @include flex($flexDirection: column);
+      justify-content: center;
+      .logoCircleBig {
+        border: 0px solid#BEBAFF80;        ;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: double semicolons, no spaces

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+  height: 500px;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  .dataInputFields {
+      background-color: #fff;
+      height: 100%;
+      width: 100%;
+      margin: auto;
+      border-radius: 50px;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss:41]: 10px

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+        border-radius: 31px;
+        position: absolute;
+        width: 62px;
+        height: 62px;
+        top: 48px;
+        left: 279px;
+        margin-left: auto;
+        background-color: #BEBAFF80;
+        .logoCirclelittle {
+          border: 0px solid#AFAAFF99;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: space after  solid

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+      background-color: #fff;
+      height: 100%;
+      width: 100%;
+      margin: auto;
+      border-radius: 50px;
+      position: relative;
+      box-shadow: $boxBgShadow;
+      @include flex($flexDirection: column);
+      justify-content: center;
+      .logoCircleBig {
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: u may download picture

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+    width: 100%;
+    height: 15%;
+    margin-top: 4rem;
+    text-align: center;
+    & > h1 {
+      text-transform: uppercase;
+      color: $purple;
+      margin: auto;
+      height: 70px;
+      width: 341px;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: width h1 - by default must be 100%

File: client/src/pages/ForgotPasswordPage/styles.module.scss [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+      top: 187px;
```
Comment (OyakiMorse) [inline client/src/pages/ForgotPasswordPage/styles.module.scss]: instead top, left etc, if it`s no absolute elements u can use margin

**Review by OyakiMorse — CHANGES_REQUESTED** [review]

File: client/src/pages/ForgotPasswordPage/styles.module.scss (hunk 7 above)
Comment (Bagdunn) [reply client/src/pages/ForgotPasswordPage/styles.module.scss:41]: can you explain more precisely what is wrong with the radius?

**Review by Vadym342 — APPROVED** [review]

**Comment by MattMsh** [comment]
@Bagdunn, please, fix problems

File: client/src/pages/ForgotPasswordPage/ForgotPassword.jsx [hunk 12]
```diff
@@ -29,16 +43,16 @@ const ForgotPassword = () => {
...
             <Button
               variant='contained'
               type='submit'
-              className = {styles.resetButton}
+              sx={style.resetbutton}
             >
               Submit Email
             </Button>
-            <a href='/login' className={styles.loginHref}>Back to log in</a>
+            <a href='/' className={styles.loginHref}>Back to log in</a>
```
Comment (MattMsh) [inline client/src/pages/ForgotPasswordPage/ForgotPassword.jsx]: instead of the ```<a>``` tag, it's best to use ```<Link />``` from ```react-router-dom```

**Review by MattMsh — CHANGES_REQUESTED** [review]

**Review by MattMsh — APPROVED** [review]

**Review by 1Artemida — APPROVED** [review]

**Pull request merged**